app/src/types/api-models.ts

```diff
@@ -42,28 +42,27 @@ export type SendInntektsmeldingRequestDto = {
   kontaktperson: KontaktpersonDto;
   startdato: string;
   inntekt: number;
-  inntektEndringsÅrsak?: InntektEndretÅrsakDto;
-  refusjonsperioder: RefusjonsperiodeRequestDto[];
-  bortfaltNaturaltytelsePerioder: NaturalytelseRequestDto[];
+  refusjon?: number;
+  refusjonEndringer: RefusjonsEndringRequestDto[];
```

Comment:
Ser at her bruker vi både refusjonEndringer og RefusjonsEndring (med og uten genitiv s) – jeg liker kanskje refusjonsendringer (med s og liten e i endringer) best personlig – hva synes du?

Reply:
Enig

Reply:
Men det er litt ymse i backend, så burde ta en sync på det

Reply:
Tenker vi går for `refusjonsendring`. Så gjør jeg en backend refactor på det også